test/e2e/framework/config/config.go

```diff
@@ -38,10 +38,9 @@ type Config struct {
 	// require access to repo files, such as Helm charts and test fixtures.
 	RepoRoot string
 
-	Ginkgo    Ginkgo
-	Framework Framework
```

Comment:
So it looks like `Config.Framework` was never assigned and we had been calling methods on a nil pointer.